qiskit/circuit/quantumcircuit.py

```diff
@@ -3659,7 +3695,6 @@ def clear(self) -> None:
                 quantum and classical typed data, but without mutating the original circuit.
         """
         self._data.clear()
-        self._parameter_table.clear()
         # Repopulate the parameter table with any phase symbols.
         self.global_phase = self.global_phase
```

Comment:
Is this still necessary, or does it get done during the `CircuitData` constructor?

Reply:
Maybe, I don't remember. Let me try removing it and see what happens.